testsuite/src/test/java/org/jboss/intersmash/testsuite/provision/openshift/OpenShiftProvisionerTestBase.java

```diff
@@ -143,6 +143,13 @@ public String eeFeaturePackLocation() {
 				return IntersmashConfig.getWildflyEeFeaturePackLocation();
 			}
 
+			@Override
+			public String featurePackLocation() {
+				// this value is supposed to be overridden by the CI Jenkins job by passing e.g.
```

Comment:
Could we please change this comment (here and in other similar occurrences) and avoid mentioning details?
E.g.: I'd do: `s/overridden by the CI Jenkins job/overridden externally/`

Reply:
@fabiobrz  done